pkg/rebalancer/storage/controller/service/rebalancer.go

```diff
@@ -455,10 +448,10 @@ func (r *rebalancer) genPodModels() (podModels map[string][]*model.Pod, err erro
 	return
 }
 
-func (r *rebalancer) namespaceByJobs() (jobmap map[string][]job.Job, err error) {
+func (r *rebalancer) jobsByNamespace() (jobmap map[string][]job.Job, err error) {
```

Comment:
How about it?
We can define a more meaningful and understandable function name.

```suggestion
func (r *rebalancer) getJobsforNs() (jobmap map[string][]job.Job, err error) {
```
or
```suggestion
func (r *rebalancer) getJobsByNs() (jobmap map[string][]job.Job, err error) {
```

Reply:
Got it.
I think `getJobsByNs ` is good!
but the abbreviations are confusing, so Is it okay to use `getJobsByNamespace`? :bow:

Reply:
thank you
I fixed: https://github.com/vdaas/vald/pull/1187/commits/1be6a01131f9e1464c2b404f71b93ea7de8d90e4